beacon-chain/core/helpers/committee.go

```diff
@@ -175,6 +175,17 @@ func AttestingIndices(state *pb.BeaconState, data *ethpb.AttestationData, bf bit
 	return indices, nil
 }
 
+// VerifyBitfield validates a bitfield with a given committee size.
+func VerifyBitfield(bf bitfield.Bitfield, committeeSize uint64) error {
```

Comment:
this isnt verifying the bitfield, its just checking the length of the bitfield. Maybe change the naming to reflect this more

Reply:
`VerifyBitfieldLength`